src/styles/blocks/stars.scss

```
@@ -0,0 +1,35 @@
+.card__reviews {
+  margin-top: 16px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.card__label {
+  text-align: right;
+  padding-left: 17px;
+  font-size: 10px;
+  color: $color_text_main;
+  line-height: 18px;
+}
+
+
+.card__stars {
```

```suggestion
stars {
```
it's a separate BEM block

You **can** use BEM block inside another block